worker/sign_up_for_mofo_newsletter.js

```diff
@@ -0,0 +1,12 @@
+var httpRequest = require('request');
+var EVENT_TYPE = "sign_up_for_mofo_newsletter";
+
+module.exports = function(event, cb) {
```

Comment:
So, this is a pretty generic request maker. Could you make it more specific to basket *or* make it more generic so that it could shoot off any http request?